components/elm/src/main/subgridMod.F90

```diff
@@ -9,7 +9,7 @@ module subgridMod
   use spmdMod     , only : masterproc
   use abortutils  , only : endrun
   use elm_varctl  , only : iulog
-
+  use GridcellType     , only : grc_pp
   implicit none
```

Comment:
Please check - I think grc_pp is not used

Reply:
@thorntonpe yes, you are correct. I have removed the insertion in my latest commit. Thanks.